src/register/containers/hmcts-form-builder/src/lib/components/jui-form-elements/jui-form-elements.component.html

```diff
@@ -73,6 +73,13 @@
                       [labelFor]="data.textarea.control"
                       [rows]="5">
       </app-textareas>
+
+      <app-extensions *ngIf="data.extension"
+                        [title]="data.extension.componentDetails.title"
+                        [text]="data.extension.componentDetails.text"
```

Comment:
i think here i would pass just one object and to the distracting in the components